### DISCUSSION TITLE
wstring characters are trimmed in Linux

### DISCUSSION THREAD

**Discussion by kothapraveen** [swig/swig#2989] [General]
Hi

with swig tool we should  able to  convert from CPP to C# successfully in WINDOWs and verified also its working fine.
but in **Linux we are facing issue with WIDE strings - alternate characters are trimming and could observe junk characters also** but working fine for normal strings.

please suggest us some inputs to resolve this issue

Regards,
Praveen

**Comment by ojwb** [comment]
Please don't @-mention a random developer when you open an issue or discussion.  Thanks.
